integration/spark/app/src/main/java/io/openlineage/spark/agent/OpenLineageSparkListener.java

```diff
@@ -174,7 +175,7 @@ public void onJobStart(SparkListenerJobStart jobStart) {
             .map(Integer.class::cast)
             .collect(Collectors.toSet());
 
-    if (sparkVersion.startsWith("3")) {
+    if (SPARK_VERSION_3.compareTo(sparkVersion) < 0) {
```

Comment:
Is this really more clear? Does not seem to me...

Reply:
moving this to `SparkVersionsUtils`